packages/commons/src/fixtures/tennis-club-membership-event.ts

```diff
@@ -669,7 +669,8 @@ export const TENNIS_CLUB_FORM = defineForm({
   review: {
     title: {
       id: 'v2.event.tennis-club-membership.action.declare.form.review.title',
-      defaultMessage: 'Member declaration for {firstname} {surname}',
+      defaultMessage:
+        'Member declaration{hasAnyKey, select, true { for {applicant.firstname} {applicant.surname}} other {}}',
```

Comment:
The ICU Message syntax is super limited but I still wonder if there was a way of doing this without introducing a custom variable. Mostly just to not have to document the variable anywhere 😄

Reply:
If we agreed to use something like `__EMPTY__` to represent non-existing values in the localisation strings, then the expression could be this:

```
{firstname, select, 
    __EMPTY__ {Member declaration}
    other {{surname, select, 
        __EMPTY__ {Member declaration}
        other {Member declaration for {firstname} {surname}}
    }}
}
```

You can test this here: https://localizely.com/icu-message-editor/

Reply:
Yeah I was also trying to figure out a way without introducing a custom variable, but didn't figure out anything better.

I guess in your example we would also need to document the `__EMPTY__` expression somewhere, so I guess it has the same problem 🤔

Reply:
Sounds good 👍